lib/hash.h

```diff
@@ -236,6 +238,8 @@ extern void *hash_release(struct hash *hash, void *data);
  * during the walk will cause undefined behavior in that some new entries
  * will be walked and some will not.  So do not do this.
  *
+ * The backet passed to func will not have a non-NULL data pointer.
```

Comment:
Shouldn't this be `will have a non-NULL`?

Reply:
english hard, fixed